Extension/tools/GenerateOptionsSchema.ts

```diff
@@ -64,7 +66,7 @@ function refReplace(definitions: any, ref: any): any {
 
     // Retrieve ObjectName from definitions. (TODO: Does not retrieve inner objects)
     // Need to deep copy, there are no functions in these objects.
-    const reference: any = JSON.parse(JSON.stringify(definitions[referenceName]));
+    const reference: any = JSON.parse(JSON.stringify(definitions[referenceName],null,4));
```

Comment:
We always (?) use spaces after `,`.

Reply:
This one doesn't need it an update though because the output is immediately parsed again rather than written to disk.  Only the update on line 137 is needed.